### DISCUSSION TITLE
How to install libjxl on Windows?

### DISCUSSION THREAD

**Discussion by ChrisX4Ever** [libjxl/libjxl#3799] [Q&A]
I have the intention to install JPEG-XL Codec on my Windows 11 Notebook, but I didn't understand how to do it. I downloaded the "jxl-x64-windows-static.zip" file but I'm not sure how to install it.

Can somebody please help a noob here? Thanks in advance.

**Comment by jonnyawsom3** [comment]
Answered on the Discord server
